apps/k8s-io/configmap-nginx.yaml

```diff
@@ -306,7 +306,8 @@ data:
           rewrite ^/stuck-prs$       https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Algtm%20label%3Aapproved%20-label%3Ado-not-merge%20-label%3Aneeds-rebase%20sort%3Aupdated-asc%20-status%3Asuccess redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-triage/index.html redirect;
-          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;
+          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;          rewrite ^/contact$         https://github.com/kubernetes/community/tree/master/communication/ redirect;
+          rewrite ^/contact/([^/]*)/?$ https://github.com/kubernetes/community/tree/master/$1#contact redirect;
```

Comment:
I think you removed the wrong line?
`/contact` is now missing

Reply:
resolved